src/install/redis/host/percona.mdx

```diff
@@ -0,0 +1,31 @@
+---
+componentType: default
+headingText: Prepare Redis or Percona
```

Comment:
I wonder if the "or Percona" aspect of this integration should be introduced in your intro. I'm not sure how these relate (does Percona fall under Redis? Are they parallel 'things' that are mutually exclusive?) Depending on that, maybe the title needs rework too